buildSrc/src/main/kotlin/ort-base-conventions.gradle.kts

```diff
@@ -44,6 +44,15 @@ repositories {
             includeGroup("org.gradle")
         }
     }
+
+    maven {
+        // com.blackducksoftware.bdio:bdio2
+        url = uri("https://sig-repo.synopsys.com/bds-bdio-release")
+    }
+
+    maven { // com.blackducksoftware.magpie:magpie
```

Comment:
Please use the `exclusiveContent` to limit the use of these repos to the configured `includeGroup`. Then you also don't need the comments, as it's documented by code.